Unable to run any workflow

First of all, I don't have an ideal setup for this, because I run an rpm-ostree based system (Fedora Kinoite) and in order to use podman-compose I had to unlock my deployment and install it. Might need to overlay it for the future runs.

Anyway, I tried to follow the steps and successfully ran `make build` that created the images:
```
localhost/goose-container                  latest      8b4e9a4fbed5  About an hour ago  1.09 GB
localhost/testing-farm-sse-bridge          latest      6b7905a662a1  About an hour ago  285 MB
```
Then I tried to continue with `make run-goose`, but it fails on:
```
[jgrulich@fedora ai-workflows (main)]$ make run-goose
podman compose up -d goose
>>>> Executing external compose provider "/usr/bin/podman-compose". Please see podman-compose(1) for how to disable this message. <<<<

d8c93c6c38dfbb2df39b6a0d08a8b536a23f2d6f0171e68e57ced1428606b94e
5b5e430d99f2d6457c67f593a1b9fd0ab32e3dbff59423074f34857782702bfa
ai-workflows_mcp-atlassian_1
03fcf23ed60b9e317b55aab34629d46ca9f1046e5d28c2785d447441c352626f
ai-workflows_mcp-testing-farm_1
79bd012f77b183c4ca9a7417309e249df1a3c72372c32264682f2261c7678992
Error: unable to start container "79bd012f77b183c4ca9a7417309e249df1a3c72372c32264682f2261c7678992": crun: executable file `/usr/local/bin/goose` not found: No such file or directory: OCI runtime attempted to invoke a command that was not found
Error: executing /usr/bin/podman-compose up -d goose: exit status 125
make: [Makefile:85: run-goose] Chyba 125 (ignorováno)
podman compose exec goose goose
>>>> Executing external compose provider "/usr/bin/podman-compose". Please see podman-compose(1) for how to disable this message. <<<<

Error: can only create exec sessions on running containers: container state improper
Error: executing /usr/bin/podman-compose exec goose goose: exit status 255
```
It looks like `goose` is not properly installed or perhaps not installed at all, because when I created a container from the `goose-container` image, overriding the entry point so I can create it, I couldn't find `goose`. 

Another thing I tried was to run e.g. `make triage-issue`, but that didn't work either and failed with:
```
[jgrulich@fedora ai-workflows (main)]$ make triage-issue
podman compose run --rm \
        --entrypoint /bin/sh goose \
        -c "/home/goose/wait_mcp_server.sh && /usr/local/bin/goose run --recipe recipes/triage-issue.yaml \
                --params issue=RHEL-78418"
>>>> Executing external compose provider "/usr/bin/podman-compose". Please see podman-compose(1) for how to disable this message. <<<<

2c27857b5d21ac37acfbe3dd445643f36be87366b5180ce021a29e0ca422f96a
9ef31d13bfefa70d3a383acbc5c9fc74fd37aaf3278bfca74d74f68fac5ff0a0
ai-workflows_mcp-atlassian_1
8db9a58bc15490362c384aad132b590860d68b6d256ee4ac2cb4cfd2c4b37513
ai-workflows_mcp-testing-farm_1
/bin/sh: line 1: /home/goose/wait_mcp_server.sh: No such file or directory
Error: executing /usr/bin/podman-compose run --rm --entrypoint /bin/sh goose -c /home/goose/wait_mcp_server.sh && /usr/local/bin/goose run --recipe recipes/triage-issue.yaml               --params issue=RHEL-78418: exit status 127
make: *** [Makefile:106: triage-issue] Chyba 127
```
Btw. I had to run `make clean` first as I assume when `make run-goose` failed, it didn't properly cleaned up the containers. This I think happens because the `Compose.yaml` file has:
```
    volumes:
      - ./agents:/home/adk/agents:ro,z
      - ./wait_mcp_server.sh:/wait_mcp_server.sh:ro,z
```
Which I think (I'm not really an expert here) installs the `wait_mcp_server.sh` script to the root directory instead.

Maybe I'm doing something wrong?

--- a/goose-container/Containerfile
+++ b/goose-container/Containerfile
@@ -58,6 +58,8 @@ FROM runtime-platform AS production
 
 USER root
 
+ARG GOOSE_VERSION
+
 RUN curl -fsSL https://github.com/block/goose/releases/download/${GOOSE_VERSION}/download_cli.sh | \
     GOOSE_BIN_DIR=/usr/local/bin CONFIGURE=false bash
 

